packages/builtin-microsoft/src/builtin-microsoft.js

```diff
@@ -319,12 +319,13 @@ class BuiltinMicrosoft extends Clonable {
         }
         for (let i = 0; i < entities.length; i += 1) {
           const entity = entities[i];
-          source.push(entity);
           let entityName = entity.typeName;
           const index = entityName.lastIndexOf('.');
           if (index !== -1) {
             entityName = entityName.slice(index + 1);
           }
+          entity.entity = entityName;
```

Comment:
Maybe can be useful keeping track of the raw entity name provided by microsoft. What do you think of adding rawEntity field keeping that info? from a developer point of view, can be somehow useful?

Reply:
Yes, but then - for consistency reasons - I also add it for duckling